tests/lambda_endpoints/test_query_api.py

```diff
@@ -59,18 +59,18 @@ def expected_response():
     expected_response = json.dumps(
         str(
```

Comment:
I think this might be an issue with the `str(` call here that I was running into yesterday. Can you remove that and let json dump the objects directly to string?

```
>>> json.dumps([1, 2, 3])
'[1, 2, 3]'
>>> json.dumps(str([1, 2, 3]))
'"[1, 2, 3]"'
```

Reply:
Unfortunately this wouldn't work with `datetime` objects:

```
 TypeError: Object of type datetime is not JSON serializable
```

Reply:
Hm... Can we change those datetime objects to strings somewhere along the path before returning? What do end users need, the start_date and end_date as YYYYMMDD string is all, or do we need the full datetime object printed out?

Reply:
Thanks for the suggestion, I got things working this way.